# Literature

## Deep analysis of neuroblastoma core regulatory circuitries using online databases and integrated bioinformatics shows their pan-cancer roles as prognostic predictors

*  Published: 29 November 2021

*  NB consists of dual cell identities (adrenergic and mesenchymal), which are conferred by a small set of tightly-regulated transcription factors (TFs) binding super enhancers, collectively forming core regulatory circuitries (CRCs)

*  Side note: pan-cancer means assessing things (like mutations) across several cancers regardless of tumor origin

*  NB is formed of cells of the neural crest that are halted in their developmental stages and fail to differentiate

*  Tumours are classified by the prevalence of neural crest migratory (mesenchymal, MES) and committed adrenergic (ADRN) cellular sub-types

*  ADRN subtype-specific genes include PHOX2B, PHOX2A, and DBH, while the MES subtype expresses high levels of SNAI2, FN1, and VIM

*  long non-coding RNAs (lncRNAs) are often upregulated in cancer and are increasingly characterised as potential prognostic and diagnostic biomarkers and therapeutic targets for several cancers, including NB

*  In NB, T-cell infiltration to the tumour microenvironment (TME), correlates with enhanced patient overall survival (OS)

*  Specifically, the expression and activation of MYCN, ASCL1, and SOX11 are inversely correlated with T-cell infiltration in NB and correlate with patient OS, supporting the significance of these TFs in the NB TME

*  For PANTHER GO, they used the browser, entered the MES or ADRN genes list in the text box, selected Homo sapiens in box 2, Statistical overrepresentation test in box 3 (selected PANTHER pathways when drop-down shows up), Submit, Default whole-genome lists on the right was Homo sapiens (submits automatically), Launch analysis

*  Narrowed our analysis to the 20 MES and 18 ADRNs TFs, as reported by van Groningen and colleagues, since these TFs comprise the core defining factors of MES and ADRN identities and include ELK4, CREG1, DCAF6, ID1, SMAD3, SIX4, SIX1, MAML2, NOTCH2, CBFB, IFI16, ZNF217, EGR3, ZFP36L1, WWTR1, PRRX1, SOX9, MEOX1, MEOX2, AEBP1, for MES and ZNF536, PHOX2A, HAND1, ASCL1, KLF13, SOX11, GATA2, GATA3, KLF7, EYA1, TFAP2B, ISL1, HEY1, SIX3, DACH1, PHOX2B, PBX3, SATB1 for ADRN

*  list of genes involved in EMT and cancer stemness markers including CD44, CDH1, CDH2, FN1, FOXC2, NANOG, SOX2, TWIST1, and VIM

*  Gene Ontology analysis was conducted on the 485 MES and 369 ADRN genes previously reported to define the respective NB cell subtypes

*  Analysis of ADRN genes yielded the term ‘enrichment for adrenaline and dopamine-related pathways’

*  For MES, terms such as ‘angiogenesis’, ‘cadherin’, ‘PDGF’, ‘integrin’, ‘JAK/STAT signalling pathways’, and ‘inflammatory pathways’ were obtained

*  Upon correlating the refined list of 38 TFs with risk groups in NB patient samples, we report that 8 ADRN and 8 MES TFs were significantly up- or downregulated in high-risk NB

*  These TFs include, for the ADRN group, SATB1, GATA2, TFAP2B, KLF13, KLF7, and PBX3 that were downregulated in the NB high-risk group (p<0.0001, 0.0002, 0.0001, 0.0141, 0.0002, and 0.028, respectively).

*  Conversely, SIX3 and GATA3 were upregulated in this group (p = 0.0133, not significant, respectively)

*  Among the MES genes, MEOX1, CBFB and DCAF6 were downregulated, (p = 0.046, 0.0001 and 0.0001, respectively), while SMAD3, ID1, SOX11, ZNF217 and EGR3 were upregulated in NB high-risk groups (p = 0.009, 0.0115, 0.014, 0.021 and 0.035 respectively)

*  Therefore, these TFs (isolated or in clusters) might represent biomarkers for assigning patients to risk groups for treatment stratification

*  KLF7 and TFAP2B were upregulated in patients with increased survival.

*  In contrast, HAND1, EGR3, PBX3, and ASCL1 were upregulated in patients with reduced survival (although both TFAP2B and HAND1 show trends)

*  Notably, KLF7 was associated with better survival outcomes based on both RNA sequencing and microarray analyses

*  These data suggest a significant association of expression of these TFs with NB high-risk groups and their association with patient survival outcomes

*  However, these data will need to be validated in preclinical models and in a prospective clinical trial scenario, whereby a predictive outcome algorithm can be developed

*  The correlation analyses of Spearman and Pearson found lncRNAs that positively correlate with MES and ADRN CRC genes

*  For instance, MEOX2, SIX1, GATA2, TFAP2B, GATA3, SIX3, PHOX2A, GATA2, SATB1, and TFAP2B expression positively correlate with LINC02587, EMX2OS, DBH-AS1, DBH-AS1, GATA3-AS1, SIX3-AS1, MORC2-AS1, GATA2-AS1, KLF9-AS1, and LIFR-AS1, respectively

*  Notably, DBH-AS1 expression strongly correlates with GATA2 (Spearman = 0.64, p = 2.59E-17) and TFAPB2 (Spearman = 0.72, p = 6.17E−24)

*  Side note: cbioportal is really useful and has TARGET in it for various things and even has an api

*  lncRNAs can interact with TFs, mostly by stabilising them and promoting their downstream activity

*  GEPIA2: useful for looking at the expression of genes of interest across numerous cancers (pan-cancer)

---

## Neuroblastoma is composed of two super-enhancer-associated differentiation states

*  Published: 26 June 2017

*  


















